samples/cpp/benchmark_app/main.cpp

```diff
@@ -35,6 +35,41 @@
 #include "remote_tensors_filling.hpp"
 #include "statistics_report.hpp"
 #include "utils.hpp"
+
+#if defined _WIN32
+
+#include <windows.h>
+#include <psapi.h>
+
+int64_t getPeakMemoryUsage() {
+    PROCESS_MEMORY_COUNTERS memCounters;
+    GetProcessMemoryInfo(GetCurrentProcess(), &memCounters, sizeof(memCounters));
```

Comment:
I see multiple places with the same utility methods: [ex1](https://github.com/openvinotoolkit/openvino/blob/d7d43a0dd38aa012c8c69245fa6b15c5485e9a73/src/tests/test_utils/common_test_utils/src/common_utils.cpp#L60), [ex2](https://github.com/openvinotoolkit/openvino/blob/d7d43a0dd38aa012c8c69245fa6b15c5485e9a73/tests/stress_tests/common/utils.cpp#L40), [ex3](https://github.com/openvinotoolkit/openvino/blob/d7d43a0dd38aa012c8c69245fa6b15c5485e9a73/src/plugins/intel_gpu/include/intel_gpu/runtime/profiling.hpp#L241)..
Please let me know if I can(should) use some of them, or move it to some common folder.

Reply:
samples cannot use utils from tests / openvino since they should be compilable based on prebuilt OV where such utils are not available

Reply:
But you can request openvino to introduce that feature.

Reply:
Am I correct in understanding that you generally agree to have these methods here?

> But you can request openvino to introduce that feature.

To me it would be better to finish this PR, I can open a request for the refactoring later if you are comfortable with that.

Reply:
I'm fine with these methods here